libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/SpecClass.hs

```diff
@@ -148,7 +150,7 @@ instance Specification Size Int where
   genFromS _ _ _ = genFromSize
 
 instance
-  (Era era, Ord dom {- HasRep dom, HasRep rng, HasRep c, -}, Ord rng, Adds rng) =>
+  (EraPParams era, Arbitrary (PParamsHKD StrictMaybe era), Ord dom {- HasRep dom, HasRep rng, HasRep c, -}, Ord rng, Adds rng) =>
```

Comment:
```suggestion
  (EraPParams era, Arbitrary (PParamsHKD StrictMaybe era), Ord dom, Ord rng, Adds rng) =>
```
I think somebody might have just accidentally left that comment in while debugging